### PR TITLE
Gslux 642 3d catalog

### PR DESCRIPTION
**Pull request by mki-c2c**

### CODE REVIEW

File: geoportal/geoportailv3_geoportal/static-ngeo/js/olcs/Lux3DManager.js [hunk 1]
```diff
@@ -137,6 +137,18 @@ const exports = class extends ngeoOlcsManager {
 
     this.mapStore_ = useMapStore();
 
+    const { layers_3d } = storeToRefs(this.mapStore_)
+    watch(
+      layers_3d,
+      (layers_3d, oldLayers_3d) => {
+        console.log(layers_3d, oldLayers_3d);
```
Comment (tkohr) [inline geoportal/geoportailv3_geoportal/static-ngeo/js/olcs/Lux3DManager.js:144]: `console.log` to be removed

**Review by tkohr — APPROVED** [review]
Works well, thanks @mki-c2c !

Two minor things to be handled here or in parent PR:
- Layerswitcher should be hidden in mesh mode
- The order of "3d layers" and 3d meshes" groups in the catalog is strangely reversed compared to v4 and prod

**Comment by mki-c2c** [comment]
> Works well, thanks @mki-c2c !
> 
> Two minor things to be handled here or in parent PR:
> 
> * Layerswitcher should be hidden in mesh mode
> * The order of "3d layers" and 3d meshes" groups in the catalog is strangely reversed compared to v4 and prod

good catch, thanks

File: geoportal/geoportailv3_geoportal/static-ngeo/js/olcs/Lux3DManager.js (hunk 1 above)
Comment (mki-c2c) [reply geoportal/geoportailv3_geoportal/static-ngeo/js/olcs/Lux3DManager.js]: OK

**Comment by mki-c2c** [comment]
> Works well, thanks @mki-c2c !
> 
> Two minor things to be handled here or in parent PR:
> 
> * Layerswitcher should be hidden in mesh mode

=> done

> * The order of "3d layers" and 3d meshes" groups in the catalog is strangely reversed compared to v4 and prod

=> this depends on the order in the DB, the dev DB has the 3d layers defined in inverse order. If you change the positions in the admin console, you will get the same order as in prod. There should be no functional change to the code.

**Pull request merged**